frontend/packages/console-shared/src/test-utils/actions.view.ts

```diff
@@ -1,10 +1,10 @@
 import { $, $$, browser, ExpectedConditions as until } from 'protractor';
 import { rowForName } from '@console/internal-integration-tests/views/crud.view';
-import { waitForCount, click } from '@console/shared/src/test-utils/utils';
-import { PAGE_LOAD_TIMEOUT_SECS } from '../tests/utils/consts';
+import { waitForCount, click } from './utils';
 
-const disabledDropdownButtons = $$('.pf-m-disabled');
+const PAGE_LOAD_TIMEOUT_SECS = 15 * 1000;
```

Comment:
not sure if it makes sense to have this passed as an optional parameter?

Reply:
I think it's not necessary, because these methods only click an action (and confirm if needed), and don't wait for an effect of the action. 
So I don't think there should be a need to change the timeout duration